kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -1105,3 +1111,23 @@ def print_node(self, kcfg: KCFG, node: KCFG.Node) -> list[str]:
                 path, start, end = srcmap_data
                 ret_strs.append(f'src: {str(path)}:{start}:{end}')
         return ret_strs
+
+
+class FoundryAPRNodePrinter(FoundryNodePrinter, APRProofNodePrinter):
```

Comment:
This diamond inheritance is not ideal, we need to figure out a way to split it somehow.